monitoring/grafana-dashboards/percona-sql/MySQL-Overview-Interleaved.json

```diff
@@ -0,0 +1,5208 @@
+{
```

Comment:
Do we really need these dashboards. I suppose the galera pxc ones are good enough?

Reply:
Yes @ksatchit we need this for a non PMM setup